src/pages/getting-started.jsx

```diff
@@ -44,6 +44,21 @@ function TechPlaybook() {
   );
 }
 
+function EsaPlaybook() {
+  return (
+    <div className="margin-bottom--lg">
+      <img src="/img/esa-playbook.jpg" alt="[ESA playbook cover image]" />
+      <Heading as="h2">Educational Service Agency (ESA) Playbook</Heading>
+      <Link
+        to="/getting-started/esa-playbook"
+        className="button button--primary button"
+      >
+        Go »
+      </Link>
+    </div>
+  );
+}
+
```

Comment:
Along with the import, I also created a function for the ESA Playbook inside of `getting-started.jsx`. Let me know if I should revert this. I also saw the refactor note inside of this file and would not mind working on that.